### PR TITLE
created description and result page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
created description and result page of quiz
Screenshot (95)
Screenshot (96)

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Review by zabihshaik — APPROVED** [review]
Everything looks good to merge.

File: jsquiz/description/description.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<body>
+    <a href="/landing-page/landing.html"><div class="logo-box col-2">
+        <div class="brand-logo">
+            <img src="/images/brand-logo.svg" alt="">
+        </div>  
+      <h1 class="brand-name">Fun<span class="brand-name-tag">Play</span></h1>
+      </div></a>
+
+       <!-- ---------------------------------------------------------------------------------- -->
+       <div class="main-row col-12">
```
Comment (pritam-kr) [inline jsquiz/description/description.html:22]: class name can be bette

File: jsquiz/result copy/result.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+width: 100%;
+/* border: 1px solid black */
+
+}
+.score{
+    border: 2px dashed black;
+    padding: 0.5rem 2rem;
+}
+.result-details-box{
+    /* border: 1px solid black; */
```
Comment (pritam-kr) [inline jsquiz/result copy/result.css:17]: removed commented code

File: money-heist/result/result.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    margin: 3rem 0 ;
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+    justify-content: center;
+    align-items: center;
+    
+}
+
+#rating-value{
```
Comment (pritam-kr) [inline money-heist/result/result.css:47]: you write CSS using class selector

File: money-heist/description/description.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        </div>
+            <div class="desc-btn">
+           <a href="/money-heist/firstquestion/first.html"><button class="money-heist-btn">PLAY</button></a>
+        </div>
+        </div>
+  
+      </div>
+
+</body>
+</html>
```
Comment (pritam-kr) [inline money-heist/description/description.html:45]: break line after the last line of code

**Pull request merged**